### PR TITLE
Fix language (typos) in Receipts.md

### PR DESCRIPTION
**Pull request by mehtaphysical**
Fix some typos in the `Receipts.md` file.

### CODE REVIEW

File: specs/RuntimeSpec/Receipts.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ Each receipt has a [`predecessor_id`](#predecessor_id) (who sent it) and [`recei
 Receipts are one of 2 types: action receipts or data receipts.
 
 Data Receipts are receipts that contains some data for some `ActionReceipt` with the same `receiver_id`.
-Data Receipts has 2 fields: the unique data identifier `data_id` and `data` the received result.
-`data` is an `Option` field and it indicates whether the result was a success or a failure. If it's `Some`, then it means
-the remote execution was successful and it contains the vector of bytes of the result.
+Data Receipts have 2 fields: the unique data identifier `data_id` and `data` the received result.
+`data` is an `Option` field and it indicates whether the result was a success or a failure. If it's `Some`, it means
+the remote execution was successful and it contains a result as a vector of bytes.
```
Comment (bowenwang1996) [inline specs/RuntimeSpec/Receipts.md:13]: "it contains a result as a vector of bytes" still feels a bit awkward. Do you think "it represents the result as a vector of bytes" would be better?

File: specs/RuntimeSpec/Receipts.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ Each `ActionReceipt` also contains fields related to data:
 Each `DataReceiver` consists of `data_id` and `receiver_id` for routing.
 
 Before any action receipt is executed, all input data dependencies need to be satisfied.
-Which means all corresponding data receipts has to be received.
-If any of the data dependencies is missing, the action receipt is postponed until all missing data dependency arrives.
+Which means all corresponding data receipts have to be received.
+If any of the data dependencies are missing, the action receipt is postponed until all missing data dependency arrive.
```
Comment (bowenwang1996) [inline specs/RuntimeSpec/Receipts.md:23]:
```suggestion
If any of the data dependencies are missing, the action receipt is postponed until all missing data dependencies arrive.
```

File: specs/RuntimeSpec/Receipts.md [hunk 3]
```diff
@@ -93,35 +93,35 @@ If smart contract finishes its execution with some value (not Promise), runtime
...
+Receipts can be generated during the execution of a [SignedTransaction](./Transactions.md#SignedTransaction) (see [example](./Scenarios/FinancialTransaction.md)) or during application of some `ActionReceipt` which contains a [`FunctionCall`](#actions) action. The result of the `FunctionCall` could be either another `ActionReceipt` or a `DataReceipt` (returned data).
 
 # Receipt Matching
 
-Runtime doesn't expect that Receipts are coming in a particular order. Each Receipt is processed individually. The goal of the `Receipt Matching` process is to match all [`ActionReceipt`s](#actionreceipt) to the corresponding [`DataReceipt`s](#datareceipt).
+The Runtime doesn't require that Receipts come in a particular order. Each Receipt is processed individually. The goal of the `Receipt Matching` process is to match all [`ActionReceipt`s](#actionreceipt) to the corresponding [`DataReceipt`s](#datareceipt).
```
Comment (bowenwang1996) [inline specs/RuntimeSpec/Receipts.md:116]: The use of article here seems inconsistent with the rest of this file.

File: specs/RuntimeSpec/Receipts.md (hunk 1 above)
Comment (mehtaphysical) [reply specs/RuntimeSpec/Receipts.md:13]: that sounds great to me

**Review by bowenwang1996 — APPROVED** [review]

**Pull request merged**